exercises/robot-name/RobotNameTest.cs

```diff
@@ -30,4 +31,20 @@ public void Can_reset_the_name()
         robot.Reset();
         Assert.NotEqual(originalName, robot.Name);
     }
+
+    [Fact(Skip = "Remove to run test")]
+    public void After_reset_the_name_is_valid() {
+        robot.Reset();
+        Assert.Matches(@"^[A-Z]{2}\d{3}$", robot.Name);
+    }
+
+    [Fact(Skip = "Remove to run test")]
+    public void Robot_names_are_unique() {
```

Comment:
Could you put the opening curly brace on the next line?

Reply:
I spend most of my time now in Java :-)

Reply:
:)